### DISCUSSION TITLE
Can i Have a workflow as event source

### DISCUSSION THREAD

**Discussion by ollisco** [argoproj/argo-events#2121] [Q&A]
I am looking into the eventflow and I don not see a clear start for being able to have a workflow trigger a sensor. 

Is this possible? 
If so, please redirect me to the right resource.

Thanks ✨

**Comment by joepke** [comment]
Have a look at this example: https://github.com/argoproj/argo-events/blob/master/examples/event-sources/resource.yaml. We use it to trigger Workflow B when Workflow A finishes.
